setup.py

```diff
@@ -9,7 +10,7 @@
 
 # Main setup command
 setup(name='opmd_viewer',
-      version='1.0',
```

Comment:
hopefully this does not cause problems if someone has already installed the current version and then goes to install the first release aka `0.1.0`